file_paths.md

```diff
@@ -339,6 +340,39 @@ The package will install a service named `puppetmaster`, create a
         puppetmaster *                    # writeable by puppetmaster
             puppetmaster.pid
 
+# pxp-agent
+
+This package executes Puppet and replies to PXP requests.
+
+## *nix
+
+    /opt/puppetlabs/pxp-agent *
+        spool                             # :spooldir
+        modules *                         # :modulesdir
+            puppet *                      # executes Puppet and provides results of Puppet runs
+
+    /var/log/puppetlabs/pxp-agent *       # :logdir
+        pxp-agent.log                     # enabled by default
+
+    /etc/puppetlabs/pxp-agent *
+        pxp-agent.cfg                     # :configfile
+        modules.d *                       # :modules_configdir
+            puppet.cfg *                  # configuration file of the `puppet` module
+
+## Windows
+
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        var *
+            spool                         # :spooldir
+            modules *                     # :modulesdir
+                puppet.bat *              # executes Puppet and provides results of Puppet runs
+            log *                         # :logdir
+                pxp-agent.log             # enabled by default
+        etc *
+            pxp-agent.cfg                 # :configfile
+            modules.d *                   # :modules_configdir
+                puppet.cfg *              # configuration file of the `puppet` module
+
```

Comment:
Thanks @parisiale for putting this together. Would you mind moving these sections into `puppet-agent` so that it's clear how the pxp-agent component is laying out files relative to other components, e.g. puppet, mcollective, etc